apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -2394,6 +2499,13 @@ def scale_resources(min_resource, max_resource, max_input_size, input_size):
                         raise ValueError(
                             f"The random sampling ran into an issue: num_sampled ({num_sampled}) != training_sample_size ({training_sample_size})"
                         )
+                # Add dependencies for normalization
+                for node in normalization_nodes:
+                    if copy_centroids_uri is not None:
+                        centroids_node.depends_on(node)
```

Comment:
Can this actually happen?  From my reading it cannot b/c we're in the `else:` case, rigth?